### DISCUSSION TITLE
Is it possible to listen for key combinations ctrl+s for example?

### DISCUSSION THREAD

**Discussion by amoktar** [livewire/livewire#7995] [1. Help] [answered]
Hello guys.

Is it possible to listen for key combinations ctrl+s for example?

Thanks

**Comment by danie-ramdhani** [comment, marked as answer]
oh yes you can:

```blade
<span x-on:keydown.ctrl.s.window.prevent="console.log($el.innerText)">
    Hi there!
</span>
```

> but why `ctrl + s` ??

**Reply by amoktar** [reply]
Thanks @danie-ramdhani , I appreciate your help
I'm using `ctrl+s` for save curren form changes
